### PR TITLE
x86_64: Exclude a few tests that fail due to run on x86_64

### PR DESCRIPTION
**Pull request by galak**
These tests hang and for now to make forward progress lets exclude
running them.  The introduction of the demand paging code is related
to these failures.

Signed-off-by: Kumar Gala <kumar.gala@linaro.org>

### CODE REVIEW

**Comment by galak** [comment]
I opened issue #31528 so we have an issue to track what's going on.

**Review by carlescufi — APPROVED** [review]
Approving to unblock, but of course not running the fatal exception test on x86-64 should be reverted before the release

**Review by pabigot — APPROVED** [review]
Needed by power management #31273

**Comment by galak** [comment]
@nashif also submitted #31529 - I leave it to him to decide which way he wants to go forward.

**Pull request closed without merging**
